### PR TITLE
chore: Dev pr template

### PR DESCRIPTION
**Pull request by marvelman3284**
Add pr templates like we talked about

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/feature.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+(Make sure your title is either: 'fix', 'chore', or 'feat' then your title. ex: `fix: add new plugin`)
```
Comment (G-Rowell) [inline .github/PULL_REQUEST_TEMPLATE/feature.md:1]: Do you think it's worth linking the conventional commits page here?

Is it worth also worth linking the contributing.md or website page?

Comment (G-Rowell) [inline .github/PULL_REQUEST_TEMPLATE/feature.md:1]: Do you think it's worth linking the conventional commits page here?

Is it worth also worth linking the contributing.md or website page?

File: .github/PULL_REQUEST_TEMPLATE/feature.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+Fixes Issue # (If it doesn't fix an issue then delete this line)
+
+Features Added:
+- Plugin Name (Add links if possible too)
+
+Reasoning:
+List why the feature is needed 
+
+Speed:
```
Comment (G-Rowell) [inline .github/PULL_REQUEST_TEMPLATE/feature.md:11]: Worth having this as optional? or maybe even....

Show the impact of the speed on NvChad if applicable:

Like a typo fix won't need speed impact.


I do really like having this in the template though, as it keeps a focus on speed.

**Review by G-Rowell — CHANGES_REQUESTED** [review]
Also consider squashing these into one commit?

**Comment by G-Rowell** [comment]
Lemme know if I'm being too picky!

File: .github/PULL_REQUEST_TEMPLATE/feature.md (hunk 2 above)
Comment (marvelman3284) [reply .github/PULL_REQUEST_TEMPLATE/feature.md:11]: Sure sounds good

File: .github/PULL_REQUEST_TEMPLATE/feature.md (hunk 1 above)
Comment (marvelman3284) [reply .github/PULL_REQUEST_TEMPLATE/feature.md:1]: Yeah I was thinking about that and was considering moving this into contributing.md, and then just simply linking to contributing.md instead

**Comment by marvelman3284** [comment]
@G-Rowell Everything should be done now, let me know if you have more changes.

**Comment by G-Rowell** [comment]
Thanks Marvel! 

Great as always!

**Pull request merged**